Automattic/jetpack
# Writing prompts: move block to production and use for answering prompts

## Proposed changes:

* Moves writing prompt block to `production`
* Uses block when starting a new post using and `?answer_prompt={prompt_id}` query param

### Other information:


## Jetpack product discussion

PT: pe7F0s-uo-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Ensure your test site is connected to WP.com
* Enable experimental blocks (`define( 'JETPACK_EXPERIMENTAL_BLOCKS', true );`)
* Sandbox public-api and apply these changes to your wpcom sandbox with `bin/jetpack-downloader test jetpack update/answer-prompt-w-block` (if not, the block header and footer will be empy, but is works, otherwise)
* Start a new post answering a specific prompt id, like `/wp-admin/post-new.php?answer_prompt=1810`
* You should see a prompt inserted into the post content
* The tags `dailyprompt` and `dailyprompt-{prompt_id}` should be added to the post
* The post meta `_jetpack_blogging_prompt_key` should be added to the post, with the prompt id
